### PR TITLE
Update Kindle support devices list

### PR DESCRIPTION
**Pull request by hwhung0111**

### CODE REVIEW

**Comment by NiLuJe** [comment]
Oh, and adding KT3 at the bottom, too ;).

**Comment by NiLuJe** [comment]
Oops, I meant KT4 at the bottom, my bad -_-".

Basically:

KT -> KT2
KT2 -> KT3

And what you just added -> KT4

**Comment by hwhung0111** [comment]
Oh, I see.

File: platform/kindle/extensions/koreader/README.txt [hunk 1]
```diff
@@ -13,13 +13,13 @@ K4 Yes
...
 PW Yes
 PW2 Yes
-KT Yes
 Voyage Yes
 PW3 Yes
 KOA Yes
 KT2 Yes
 KOA2 Yes
 PW4 Yes
+KT3 Yes
```
Comment (Frenzie) [inline platform/kindle/extensions/koreader/README.txt]: Not a Kindle owner but the grouping seems a touch random to me. If it's by date I'm still not sure it plain alphabetical wouldn't be easier to use.

Comment (hwhung0111) [reply platform/kindle/extensions/koreader/README.txt]: Agree. I think the list order in alphabetical will be easier to read.

Comment (Frenzie) [reply platform/kindle/extensions/koreader/README.txt]: Never mind, I'm contradicting @NiLuJe

**Comment by NiLuJe** [comment]
Yeah, it was by chronological order of release ;).

**Comment by Frenzie** [comment]
@NiLuJe Was this good now?

**Comment by NiLuJe** [comment]
Oh, yes, my bad, it's fine, thanks for the ping, I missed the last update when I replied by mail.

**Pull request merged**